examples/kfp/demo.py

```diff
@@ -24,15 +24,15 @@ def main():
     model = TFModel(config)
     model.generate_files()
     pipeline = KfpPipeline(model)
+    pipeline.configure_kfp_cluster()
```

Comment:
I'm open to moving this in `pipelines.py`.

Reply:
I was thinking in the constructor of `KfpPipeline` we would check if the user has a Kubeflow Pipeline they want to use already (like if their team has an existing CAIP Pipeline deployed), if not we can make the kfp cluster for them. If that is what this function is doing, then I think putting it in the constructor makes sense.